packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -495,6 +485,41 @@ export class ARRenderer extends EventDispatcher {
         });
   }
 
+  // private placeInitially(frame: XRFrame) {
```

Comment:
meant to remove this?

Reply:
Looks like you were looking at an earlier commit?